packages/auth/src/__tests__/jestScaffolding.js

```diff
@@ -5,11 +5,8 @@
 
 import { getTestDatabase, clearTestData } from '@tupaia/database';
 
-beforeAll(done => {
-  done();
-});
-
-afterAll(async done => {
-  await clearTestData(getTestDatabase());
-  done();
+afterAll(async () => {
+  const database = getTestDatabase();
+  await clearTestData(database);
+  await database.closeConnections();
```

Comment:
It seems we reached the same conclusion in parallel! 😆 https://github.com/beyondessential/tupaia/pull/1433/files